package/Dockerfile

```diff
@@ -4,4 +4,5 @@ ENV KUBECONFIG /root/.kube/config
 RUN microdnf update -y && \
     rm -rf /var/cache/yum
 COPY bin/eks-operator /usr/bin/
+USER 1001
 ENTRYPOINT ["eks-operator"]
```

Comment:
Don't you also need a PSP that specifies user isn't root ? 
https://github.com/rancher/backup-restore-operator/pull/99/files#diff-5218b12eb3a3217f30883761568497dbdeba7df6fc347fb3869c00cc04895af6R13

And speaking of that you would also need a networkPolicy, and potentially turning off default SA. Is that all coming in follow up PR ?

Reply:
We don't need a PSP for this. Kubernetes knows that the image will not run with root user after making this change.

If I understand correctly, default service accounts and network policies are applied to namespaces. None of the KEv2 operators use their own namespaces. The KEv2 operators do create their own service accounts and don't use the default in the cattle-system namespace.

Reply:
Oh, so this is taken care of for us because its already setup that way when we setup rancher. Same for PSP's, see `default-psp-rolebinding`.